### ISSUE TITLE
Remove unused packages from run image

### ISSUE THREAD

**Issue by robdimsdale** [paketo-buildpacks/jammy-full-stack#44]
The run image contains packages that are not useful and should be removed.

This is primarily [`ruby`](https://github.com/paketo-buildpacks/jammy-full-stack/blob/55f3613cf48fc16ed1cbc58cf44afb8fc475342e/stack/stack.toml#L326), but there could be other packages that should be removed, too.

**Comment by robdimsdale** [comment]
Looks like we had a transitive `python` package that has since been removed (see https://github.com/paketo-buildpacks/jammy-full-stack/issues/14#issuecomment-1679028797)

**Comment by Fauli** [comment]
Our platform runs security scans from time to time and identifies vulnerabilities in ruby code that might be related to some packages used during build. Do you still face this as well?

**Comment by robdimsdale** [comment]
@Fauli yes, this happens. In general the full stack isn't a good choice for security-conscious folks because of the wide surface area of all the included packages.

The full stack is a great way to get started, and if you need a more stripped down image (and the base stack doesn't have the necessary libraries) then we'd encourage you to maintain your own stack with exactly the libraries you need, and no more.

**Comment by mmtevelde** [comment]
[This accepted RFC](https://github.com/paketo-buildpacks/rfcs/blob/main/text/stacks/0008-jammy-full-remove-ruby.md) should be a fine basis to implement removing Ruby from the full stack, right?

**Comment by robdimsdale** [comment]
@mmtevelde yup, we should be good to remove `ruby` from the 22.04 full stack
